shopfloor_manual_product_transfer_mobile/static/wms/src/scenario/manual_product_transfer.js

```diff
@@ -130,6 +142,16 @@ const ManualProductTransfer = {
             }
             return {};
         },
+        warning: function () {
+            const data = this.state.data;
+            if (_.isEmpty(data)) {
+                return "";
+            }
+            if (!_.isEmpty(data.warning)) {
+                return data.warning;
+            }
+            return "";
```

Comment:
```suggestion
            return _.result(this.state, "data.warning", "")
```

Reply:
BTW IMO this warning should be defined only frontend side and enabled w/ a flag instead.

Reply:
I think about it, and also think about the translations which are already integrated in the backend (not there are no translations in Shopfloor but it's easier on the backend).